packages/Adapters/Moab/test/tstMoabEntityLocalMap.cpp

```diff
@@ -340,6 +340,7 @@ TEUCHOS_UNIT_TEST( MoabEntity, quad_4_test )
 
     // Test the mapping to reference frame.
     Teuchos::Array<double> ref_good_point( space_dim );
+    // This lines throws. It seems that it frees an internal Teuchos::Array.
```

Comment:
Update the comment to something like "This lines throws when run with -fsanitize=..."